src/renderer/components/WebPlatformPlayer/LiveAppInformationDrawer.js

```diff
@@ -0,0 +1,60 @@
+// @flow
+
+import React from "react";
+import styled from "styled-components";
+import { useDispatch, useSelector } from "react-redux";
+import { useTranslation } from "react-i18next";
+
+import { SideDrawer } from "~/renderer/components/SideDrawer";
+import Box from "~/renderer/components/Box";
+import { openURL } from "~/renderer/linking";
+
+import { closePlatformAppInfo } from "~/renderer/actions/UI";
+import { platformAppInfoStateSelector } from "~/renderer/reducers/UI";
+
+import Text from "../Text";
+import ExternalLink from "../ExternalLink/index";
+import AppDetails from "../Platform/AppDetails";
+
+const Divider = styled(Box)`
+  width: 420px;
```

Comment:
avoid using hard values like these prefer either % or defined theme values.
test this with different app sizes to make sure it works :+1: